javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,13 @@
+/* exported filterOutNulls */
+
+function filterOutNulls(values) {
+  var numOnly = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] !== null) {
+      numOnly.push(values[i]);
+    } else {
+      delete values[i];
+    }
```

Comment:
Same problem here, you don't want to make changes to the original. Yu can just delete this block, you don't need to do anything with the `null` values.